code/modules/reagents/reagents/medical.dm

```diff
@@ -719,6 +736,31 @@
 /datum/reagent/medicine/quickclotplus/overdose_crit_process(mob/living/L, metabolism)
 	L.blood_volume -= 20
 
+/datum/reagent/medicine/nanoblood
+	name = "Nanoblood"
+	description = "A chemical designed to massively boost the body's natural blood restoration rate. Causes fatigue and minor toxic effects."
+	color = "#CC00FF"
+	overdose_threshold = REAGENTS_OVERDOSE/5 //6u
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/5 //10u
+	scannable = TRUE
+
+/datum/reagent/medicine/nanoblood/on_mob_life(mob/living/L, metabolism)
+	L.blood_volume += 2.4
+	L.adjustToxLoss(effect_str)
+	L.adjustStaminaLoss(6*effect_str)
+	if(L.blood_volume < BLOOD_VOLUME_OKAY)
+		L.blood_volume += 2.4
+	if(L.blood_volume < BLOOD_VOLUME_BAD)
+		L.blood_volume = (BLOOD_VOLUME_BAD+1)
+		L.reagents.add_reagent(/datum/reagent/toxin,25)
+		L.AdjustSleeping(100)
```

Comment:
but why

Reply:
ood threshold (224 blood volume), it will set your bloodloss to 225 blood volume, put you to sleep for 10 seconds, and add 25 toxin to your bloodstream. You were a moron and let MORE THAN HALF your blood fall out. Be glad you aren't de

Reply:
not a justification

Reply:
I changed it cause I wanted to.

You want me to change it back, gimme a reason to, and we can discuss that.